src/resources/postcss/components/skeleton/buttons/_icon.pcss

```diff
@@ -30,59 +30,34 @@
 	 * ----------------------------------------------------------------------------- */
 
 	/* -----------------------------------------------------------------------------
-	 * Button: Icon Caret Left
-	 * ----------------------------------------------------------------------------- */
-
-	.tribe-common-c-btn-icon--caret-left {
-
-		&:before {
-			background-image: svg-inline(caret-left-secondary);
-			height: 20px;
-			width: 12px;
-		}
-
-		&:active,
-		&:hover,
-		&:focus {
-
-			&:before {
-				background-image: svg-inline(caret-left);
-			}
-		}
-
-		&:disabled {
-
-			&:before {
-				background-image: svg-inline(caret-left-disabled);
-			}
-		}
-	}
-
-	/* -----------------------------------------------------------------------------
-	 * Button: Icon Caret Right
+	 * Button: Icon Caret Left/Right
 	 * ----------------------------------------------------------------------------- */
 
+	.tribe-common-c-btn-icon--caret-left,
 	.tribe-common-c-btn-icon--caret-right {
 
-		&:before {
-			background-image: svg-inline(caret-right-secondary);
-			height: 20px;
-			width: 12px;
+		.tribe-common-c-svgicon {
```

Comment:
let's use a class that's more specific to the `.tribe-common-c-btn-icon` class, something like `.tribe-common-c-btn-icon__icon` or something similar. That way, you wouldn't have to nest it and it targets those only with the class.